src/Sylius/Bundle/ApiBundle/Resources/config/app/config.yaml

```
@@ -1,6 +1,9 @@
 # This file is part of the Sylius package.
 # (c) Paweł Jędrzejewski
 
+parameters:
+    env(SYLIUS_API_AUTHORIZATION_HEADER): Authorization
```

```suggestion
    env(SYLIUS_API_AUTHORIZATION_HEADER): Authorization
    sylius.api.authorization_header: "%env(resolve:SYLIUS_API_AUTHORIZATION_HEADER)%"
```
and we should use `sylius.api.authorization_header` instead of `"%env(resolve:SYLIUS_API_AUTHORIZATION_HEADER)%"` to simplify usage